src/pages/invoices/index.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+import LoginRequired from 'components/login-required'
+import {useViewer} from 'context/viewer-context'
+import {GetServerSideProps} from 'next'
+import {setupHttpTracing} from '@vercel/tracing-js'
+import getTracer from 'utils/honeycomb-tracer'
+import axios from 'utils/configured-axios'
+import {isEmpty} from 'lodash'
+import {format, parseISO} from 'date-fns'
+
+const Invoices: React.FunctionComponent<any> = () => {
+  const {viewer} = useViewer()
+  const [transactions, setTransactions] = React.useState([])
+  const [transactionsLoading, setTransactionsLoading] = React.useState(true)
+
+  React.useEffect(() => {
+    axios
+      .get(`${process.env.NEXT_PUBLIC_AUTH_DOMAIN}/api/v1/transactions`)
```

Comment:
load all of a user's subscription and account transactions and display them as a list